helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -81,6 +81,9 @@ spec:
             {{- $argList = append $argList "--log-file-encoder" -}}
             {{- $argList = append $argList .Values.logging.fileEncoder -}}
             {{- end -}}
+            {{- if not .Values.leaderElection.enabled -}}
```

Comment:
Define the default value in https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/values.yaml